GetDKAN/dkan
# Add dkan:// stream wrapper

For $refs to work between JSON schemas, we really need an absolute URL. This wraps the DKAN API (everything after /api/1) in a new URI scheme, so for instance the dataset schema can reference distribution with `"$ref": "dkan://metastore/schemas/distribution"`

## QA Steps

- Install a site
- Run the follow command and confirm it returns the expected JSON; you may try additional endpoints following the same pattern.

```
dktl drush php-eval "print file_get_contents('dkan://metastore/schemas/distribution');"
```
